%YAML:1.0
CameraInfo:
  -
    Name: usbcam1_l
    DevID: 1
    Width: 352
    Height: 288
    PixelFormat: ""
    NRotate90: 0
  -
    Name: usbcam1_r
    DevID: 0
    Width: 352
    Height: 288
    PixelFormat: ""
    NRotate90: 0

StereoInfo:
  -
    Name: usb_stereo
    CamL: 0
    CamR: 1
    Width: 352
    Height: 288
    StereoParam: config/ext_usbcam1_stereo3.yaml
    StereoConfig: config/ext_usbcam1.yaml

StereoConfig:
  LensType: basic

FlowStereoInfo:
  -
    We: 2
    Wd: 3
    XFilter: 1
    YFilter: 32
    XStep: 1
    YStep: 16
    ThMatch: 16
    TFilter: 5
